etw/cs/traceloggingdynamic/TraceLoggingDynamic.cs

```diff
@@ -721,7 +756,10 @@ public EventBuilder()
         /// The initial capacity of the data buffer. This must be a power of 2 in the
         /// range 4 through 65536.
         /// </param>
-        public EventBuilder(int initialMetadataBufferSize, int initialDataBufferSize)
+        public EventBuilder(

```

Comment:
Note: Add a new constructor overload that allows the encoder to be set during construction.